api/models/release.go

```diff
@@ -181,10 +181,12 @@ func (r *Release) Promote() error {
 		for _, mapping := range entry.PortMappings() {
 			proxyParam := fmt.Sprintf("%sPort%sProxyProtocol", UpperName(entry.Name), mapping.Balancer)
 
-			switch entry.Label(fmt.Sprintf("com.convox.port.%s.protocol", mapping.Container)) {
```

Comment:
Let's keep the switch here. There will be other values for this label eventually. As mentioned in the other comment don't need the backwards compatibility since it wasn't even working before.